packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -482,9 +537,27 @@ export default class RunPanelStore {
 
     @action.bound
     onUnmount() {
+        const { journal, transactions } = this.root_store;
+
         RunPanelStore.unregisterBotListeners();
         this.disposeIsSocketOpenedListener();
 
+        if (typeof this.disposeStatisticsListener === 'function') {
+            this.disposeStatisticsListener();
+        }
+
+        if (typeof this.disposeSwitchAccountListener === 'function') {
+            this.disposeSwitchAccountListener();
+        }
+
+        if (typeof journal.disposeListeners === 'function') {
+            journal.disposeListeners();
+        }
+
+        if (typeof transactions.disposeListeners === 'function') {
+            transactions.disposeListeners();
+        }
```

Comment:
Since we're 100% sure that these are functions there's no need to do the extra check here, however we should do the check for the respective listeners. Please see later comment.